File: Kattis/fizzbuzz.cpp
#include <iostream>
#include <algorithm>
#include <vector>
#include <set>
#include <map>
#include <string>
using namespace std;

int main() {
    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);

    int a, b, n;
    cin >> a >> b >> n;

    for (int i = 1; i <= n; i++) {
        if (i % a == 0 && i % b == 0)
            cout << "FizzBuzz\n";
        else if (i % a == 0)
            cout << "Fizz\n";
        else if (i % b == 0)
            cout << "Buzz\n";
        else
            cout << i << "\n";
    }
}

File: SPOJ/fizzbuzz.py
for _ in range(int(input())):
    n = int(input())
    if n % 35 == 0:
        print('fizzbuzz')
    elif n % 5 == 0:
        print('fizz')
    elif n % 7 == 0:
        print('buzz')
    else:
        print(n)